src/main/java/io/singularitynet/utils/ImageClass.java

```diff
@@ -397,6 +402,13 @@ private void uploadInternal(int level, int offsetX, int offsetY, int unpackSkipP
         }
     }
 
+    public ByteBuffer getImageAsByteBuffer() {
+        ByteBuffer res = BufferUtils.createByteBuffer(this.getHeight() * this.getWidth() * 4);
+        res.flip();
+        GlStateManager._readPixels(0, 0, this.getWidth(), this.getHeight(), this.format.toGl(), GL11.GL_UNSIGNED_BYTE, res);
```

Comment:
It looks like ImageClass is not used anymore except Format class. I think we can safely remove it and use Format from NativeImage, or just keep the copy of Format, without ImageClass.

Reply:
So, you're suggesting to remove ImageClass and extract getImageAsByteBuffer to  VideoProducerImplementation?

Reply:
Yes

Reply:
Done